ci/src/revisions/mainnet_revisions.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+import json
+import logging
+import pathlib
+import urllib.request
+
+import git_lib
```

Comment:
What's the value of separating this code into a separate file? Doesn't seem like it's being reused by some other script.
My preference would be to have a self-contained file. Both files together are still not that big I think. Alternatively, we should do proper refactoring into multiple files, since that `git_lib` file now does argument parsing and git ops, which doesn't correspond to the name.

Reply:
We do use it in multiple places but in multiple ways. I wanted to keep the same functionality for `testnet/mainnet_revisions.json` and for the new one `mainnet_canisters.json`. The big difference is that the new one requires bazel work and I found this to be the most robust way we can keep the old and add the new. 

As for the naming, I am open to renaming it since I do agree that its not very clear what it is from the name